src/App.js

```diff
@@ -1,13 +1,76 @@
 import React from 'react';
 import './App.css';
+import contacts from './contacts.json';
 
-class App extends React.Component {
+export default class App extends React.Component {
+
```

Comment:
Very nice job Thomas, your style is always on point, as matter of fact I need lessons from you and I am not joking!! The code looks nice and clean.